LandSandBoat/server
# [lua/sql] Sugaar NM

**_I affirm:_**

## What does this pull request do?

Implements [Sugaar](https://www.bg-wiki.com/ffxi/Sugaar)

NM is very simple, though both of his moves are missing. I will do them [in a separate PR](https://github.com/LandSandBoat/server/pull/5785).

## Steps to test these changes

As the PH code is unchanged, no need to test it (but I did and confirmed it works)

goto pashow_s
`!gotoname Sugaar`
`!spawnmob 17145867`
witness fight and drops

Note that since the skills only exist in sql, he won't use them via the lua logic. Simply copy any other skill lua twice and name to the proper mob_skill_names: `grim_glower` and `torpefying_charge`

![image](https://github.com/LandSandBoat/server/assets/131182600/ce4400f3-2a46-4347-b4ec-579a2aea9be4)
